Document AI inference appliance setup on Ubuntu Core (#349)
canonical/ubuntu-core-docs: docs/tutorials/try-pre-built-images/install-on-a-vm.md

You can connect to a running instance of Ubuntu Core by opening a shell environment on that running instance:

```bash
multipass shell mycore26
```

## Create a local AI inference appliance

You can use Ubuntu Core to create a local AI inference appliance. 

The following example installs the `gemma4` inference snap and allows you to run inference directly on your Ubuntu Core VM. 

### Launch an image

Create a new instance of Ubuntu Core with enough resources to run the model, type:

```bash
multipass launch core26 -n aibox --cpus 4 --memory 10GB --disk 16GB 
```

> 

Reviewer: ````suggestion
To open a shell running within the new instance, type:

```
multipass shell aibox
```

The Ubuntu Core instance will initially update itself, and may restart automatically.

````

> **Note:**

> > Inference snaps use snap components for runtime and model weights. `gemma4` requires snapd 2.68 or newer.

Reviewer: ```suggestion
> Inference snaps use snap components for runtime and model weights. `gemma4` requires snapd 2.68 or newer, which should be installed automatically when the instance updates.
```

>
> Check your version with:
>
> ```bash
> snap version
> ```
>
> If needed, update snapd:
>
> ```bash
> sudo snap refresh snapd
> ```
> After refreshing snapd, stop and start the Ubuntu Core instance from your host machine:
>
>```bash
>multipass stop aibox
>multipass start aibox
>multipass shell aibox
>```

From inside the Ubuntu Core instance, install the snaps:

```bash
sudo snap install gemma4
```

Check the Gemma4 endpoint:

```bash
gemma4 status
```

The output includes the active engine and endpoints. For example:

```text
engine: cpu
services:
    server: active
    server-webui: active
endpoints:
    openai: http://localhost:8336/v1
    webui: http://localhost:8337/
```
The `localhost` address here is local to the Ubuntu Core instance.

## Make WebUI available from the host

By default, WebUI may listen only on the Ubuntu Core instance's local loopback address. To open it from a browser on your host machine, configure the WebUI to listen on the instance network interface:

```bash
sudo gemma4 set http.host=0.0.0.0 webui.http.host=0.0.0.0 --assume-yes
```

> From a terminal on your host machine, find the instance IP address:

Reviewer: ```suggestion
From a terminal on your host machine, outside Multipass, find the instance IP address:
```

```bash
multipass info aibox
```

The output includes an IPv4 address:

```text
Name:           aibox
State:          Running
Snapshots:      0
IPv4:           10.100.120.150
Release:        Ubuntu Core 26
```

> Open the following address in a browser on your host machine, replacing `<instance-ip>` with the IPv4 address from `multipass info`:

Reviewer: ```suggestion
Now open the instance IPv4 address on port `8337` in a web browser on your host machine.
```
